packages/realm/src/Realm.ts

```diff
@@ -1467,12 +1531,78 @@ export declare namespace Realm {
   /** @deprecated Please use named imports */
   // eslint-disable-next-line @typescript-eslint/no-namespace
   export namespace Services {
-    // TODO: Fill in once the MongoDB client has stabilized
-
+    /** @deprecated Please use named imports */
+    export type MongoDB = MongoDBType;
```

Comment:
`MongoDB` existed both as an interface and as a namespace here on the `Service` namespace.